tests/dubbo-java/src/main/java/org/apache/dubbo/samples/provider/UserProviderImpl.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dubbo.samples.provider;
+
+import org.apache.dubbo.samples.api.UserProvider;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class UserProviderImpl implements UserProvider {
+    @Override
+    public int EchoInt(int req) throws Exception {
+        return 0;
+    }
+
+    @Override
+    public byte EchoInt8(byte req) throws Exception {
+        return 0;
+    }
+
+    @Override
+    public boolean EchoBool(boolean req) throws Exception {
+        return false;
+    }
+
+    @Override
+    public Byte EchoByte(Byte req) throws Exception {
+        return null;
+    }
+
+    @Override
+    public Short EchoInt16(Short req) throws Exception {
+        return null;
```

Comment:
Waiting for implementation so just return null for now?

Reply:
I would implement provider part in next commits.